### PR TITLE
Update default colors to 18F Web Design Standards

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
- **cf-buttons:** [MINOR] Update default colors to 18F colors in Web
  Design Standards
- **cf-core:** [MINOR] Update default colors to 18F colors in Web Design
  Standards
- **cf-expandables:** [MINOR] Update default colors to 18F colors in Web
  Design Standards
- **cf-forms:** [MINOR] Update default colors to 18F colors in Web
  Design Standards
- **cf-icons:** [MINOR] Update default colors to 18F colors in Web
  Design Standards
- **cf-pagination** [MINOR] Update default colors to 18F colors in Web
  Design Standards
- **cf-tables:** [PATCH] Update link to 18F Web Design Standards
- **cf-typography:** [PATCH] Update link to 18F Web Design Standards

## Review
- @contolini (Did I do the versioning right?)

### CODE REVIEW

**Comment by contolini** [comment]
Thanks for this update! 👍 

This PR looks great but please undo all the changes you made to the components' `package.json`s. We used to manually update them but we don't anymore. CFPBot will read the CHANGELOG and make the necessary bump based on `[MAJOR|MINOR|PATCH]`.

I'll add a note about this in the contributing docs.

**Comment by KimberlyMunoz** [comment]
Great, just bumped the versions back down.

File: src/cf-buttons/src/cf-buttons.less [hunk 1]
```diff
@@ -13,24 +13,26 @@
...
+// https://github.com/18F/web-design-standards/blob/18f-pages-staging/src/stylesheets/core/_variables.scss
 
 // .btn
-@btn-text:                      #fff;
-@btn-bg:                        #c7336e;
-@btn-bg-hover:                  darken(@btn-bg, 10%);
-@btn-bg-active:                 darken(@btn-bg, 15%);
+@btn-text:                      #ffffff; // $color-white
+@btn-bg:                        #0071bc; // $color-primary
+@btn-bg-hover:                  #205493; // $color-primary-dark
```
Comment (contolini) [inline src/cf-buttons/src/cf-buttons.less]: `$color-primary-dark` should be `$color-primary-darker`

Comment (KimberlyMunoz) [inline src/cf-buttons/src/cf-buttons.less]: Thanks, fixed!

**Pull request merged**